models/engine/db_storage.py

```diff
@@ -74,3 +74,17 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Returns the object based on the class and its ID"""
+        objects = self.all(cls).values()
+        object_id = [item for item in objects if item.id == id][0]
```

Comment:
I am watching you add the index of the list comprehension before the statement and that is going to launch an error. Please check when the Id doesn't exist.

Reply:
I already changed the index before the statement